org.eclipse.winery.topologymodeler.ui/src/app/canvas/entities-modal/entities-modal.component.html

```diff
@@ -109,6 +122,49 @@
                             Will be created.
                         </div>
                     </div>
+
+                </div>
+                <!-- for policies -->
+                <div class="form-group-grouping"
```

Comment:
I thought it was more elegant to make a single *ngIf at the top of this construct depending on the case rather than having many *ngIf and ternary statements for every single HTML element in those ~50LOC that follow.

In the management ui I don't see a case for "createPolicy" in the current version:
![image](https://user-images.githubusercontent.com/12116471/45355241-2e4dd080-b5c0-11e8-8ec8-a720ad03a294.png)

Reply:
No, you don't need to have so many ifs. You can just use generic constructs which can represent both elements. In the component you can then use ONE if to determine whether you are editing a Policy or Artifact.